devlib/platform/gem5.py

```diff
@@ -127,6 +129,11 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Enable power modelling if required
+            if self.power_models_dir:
```

Comment:
You might want to put a comment in that exact argument needed to 'activate' the gem5 power models will depend on the user's implementation of it and so the user might need to change this in order for it to work. Some gem5 configuration scripts do no use "--power-model" to enable power models, for example.